contributing.md

```diff
@@ -43,8 +43,19 @@ here](https://join.slack.com/t/bazelbuild/shared_invite/enqtndyyodaxndy4mjqxlwu1
    takes too long (e.g. reviewer is unresponsive), please also send an email to
    [bazel-dev@googlegroups.com](mailto:bazel-dev@googlegroups.com).
 1. A Bazel maintainer at Google will apply the patch to our internal version
-   control system. The patch is exported as a Git commit, at which point the
-   GitHub pull request is closed.
+   control system.
+
+   This triggers internal presubmit checks like
+   [code linters](https://en.wikipedia.org/wiki/Lint_(software)), which may
```

Comment:
nit: "like code linters and internal integration tests (that will eventually be open-sourced)" or something like that? Code linters implies that they only validate style... but we have many internal tests that can reveal more serious issues -- some of which are impossible for the PR author to fix without access to those tests :-(

Reply:
With regards to "(that will eventually be open-sourced)", are efforts actually underway to open source more of the internal tests? From my perspective, Bazel has been at best treading water in this area. Consider bazelbuild/bazel@4ca2b6d5c4d201009ff21a3abbeddd9d58d4c95b.

Reply:
Re: closed tests, as I understand what remains internal remains that way because the infrastructure it depends on is too intertwined right now with Google code. So the work to untangle is higher compared to what's already been open.

Practically we should escalate this discussion to bazel-discuss or bazel-dev to try to give this more of a nudge. I really can't say how this fits in vs. all the other priorities being balanced.